.github/workflows/build-nightly.yaml

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
You should try adding in this action to make sure the wheel/build is being created properly and with the correct name:

https://github.com/neuralmagic/nm-actions/blob/main/actions/pypi_build/action.yaml

Example: https://github.com/neuralmagic/sparseml/blob/fcf3c770f88a9de028901cf4c0bcc611dc5e1875/.github/workflows/build-wheel.yml#L54